src/core/classification/qgsclassificationmethod.h

```diff
@@ -335,7 +336,7 @@ class CORE_EXPORT QgsClassificationMethod SIP_ABSTRACT
      * The maximum value is expected to be added at the end of the list, but not the minimum
      */
     virtual QList<double> calculateBreaks( double &minimum, double &maximum,
-                                           const QList<double> &values, int nclasses ) = 0;
+                                           const QList<double> &values, int nclasses, QString *error = nullptr ) = 0;
```

Comment:
```suggestion
                                           const QList<double> &values, int nclasses, QString *error SIP_PYARGREMOVE = nullptr ) = 0;
```

For the same reason as above... although, I'm unsure how sip will handle this with a virtual method :scream: